zone/attack.cpp

```diff
@@ -2764,31 +2764,31 @@ bool NPC::Death(Mob* killer_mob, int64 damage, uint16 spell, EQ::skills::SkillTy
 						}
 
 						switch (r->GetLootType()) {
-						case 0:
```

Comment:
What happened to 0

Reply:
It's unused, client doesn't have an option `0`, only `1`, `2`, `3`, and `4`.